lib/generators/tokenable/verifier_generator.rb

```diff
@@ -0,0 +1,33 @@
+# frozen_string_literal: true
+
+require 'rails/generators/active_record'
+
+module Tokenable
```

Comment:
Should the path be `lib/tokenable/generators` to match the file name path?

Reply:
Na, it needs the sub folder for the namespace. Will play around and see if i can get it to work without.